bokehjs/src/coffee/core/layout/layout_canvas.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+
+{Variable, EQ, GE, Strength}  = require "./solver"
+Model = require "../../model"
+p = require "../properties"
+
+class LayoutCanvas extends Model
+  type: 'LayoutCanvas'
+
+  @internal {
```

Comment:
This is done by adding `serializable_in_document: () -> false`. However, if you need document attached to instances of this class, then this isn't going to work. Actually, ignore this for now.